lib/mayaUsd/render/vp2RenderDelegate/material.cpp

```diff
@@ -306,6 +312,8 @@ void CMPrefs::Refresh()
         // The color management settings are quietly reset on file new:
         _mayaColorManagementCallbackIds.push_back(
             MSceneMessage::addCallback(MSceneMessage::kBeforeNew, colorManagementRefreshCB, this));
+        _mayaColorManagementCallbackIds.push_back(
+            MSceneMessage::addCallback(MSceneMessage::kBeforeOpen, colorManagementRefreshCB, this));
```

Comment:
My interactive tests showed that alternating between scenes with differing OCIO settings were giving incorrect results. So added a kBeforeOpen since OCIO will not send notifs if opening a scene results in a new config.